common/js/src/executable-module/emscripten-module.js

```diff
@@ -285,11 +279,12 @@ EmscriptenModuleMessageChannel.prototype.onMessageFromModule = function(
  * @param {!Object} message
  * @private
  */
-EmscriptenModuleMessageChannel.prototype.sendNow_ = function(message) {
+sendNow_(message) {
   // Note: The method name must match the string in the GoogleSmartCardModule
   // Embind class definition in the entry_point_emscripten.cc files.
   // The method is accessed using the square bracket notation, to make sure that
   // Closure Compiler doesn't rename this method call.
   this.googleSmartCardModule_['postMessage'](message);
-};
+}
+}
```

Comment:
This looked very confusing to me (I initially suspected a typo) - maybe a comment behind the second one that this closes EmscriptenModuleMessageChannel, or a newline might help.